parser/__tests__/index.test.js

```diff
@@ -332,4 +332,42 @@ describe('trailing whitespace', () => {
       ]);
     });
   });
+
+  describe('nested quotes', () => {
+    test('simple example', () => {
```

Comment:
Let's make this test consistent to the following one
```suggestion
    test('without whitespace between syntax', () => {
```